README.md

```diff
@@ -3,6 +3,8 @@ PHP Wrapper for Google Analytics 4 with Server Side Tracking
 [![PHP Version](https://img.shields.io/packagist/php-v/alexwestergaard/php-ga4?color=blue&style=for-the-badge)](https://www.php.net/releases/)
 [![Release Size](https://img.shields.io/github/languages/code-size/alexwestergaard/php-ga4?color=blue&style=for-the-badge)](https://github.com/AlexWestergaard/php-ga4/releases/latest)
 [![Issues](https://img.shields.io/github/issues-raw/alexwestergaard/php-ga4?color=red&style=for-the-badge)](https://github.com/AlexWestergaard/php-ga4/issues)
+[![Tests](https://github.com/AlexWestergaard/php-ga4/actions/workflows/ci.yml/badge.svg)](https://github.com/AlexWestergaard/php-ga4/actions/workflows/ci.yml)
+![Code Coverage Badge](https://raw.githubusercontent.com/AlexWestergaard/php-ga4/image-data/coverage.svg)
```

Comment:
Just a thought here.

How about making a `.github/coverage/` folder for the `coverage.svg` and then move  `https://github.com/aawnu/php-ga4/blob/master/phpunit.xml#L22` this txt file into there there too, commit it and then link it in the badge. Then people can see what falls below 100% in the coverage if need be.

I will let you decide.

Reply:
I would prefer not as I'm not sure how not existing directories are handled, I had some problems before when I put all this machinery in place. In addition, I didn't find any easy way to debug github actions.

Reply:
Roger, I will look into this :)